packages/server/src/index.ts

```diff
@@ -59,7 +59,7 @@ export const startServer = ({port}): http.Server => {
             res.status(400).json(expression);
         }
     });
-    app.use(express.static('../ui/build'));
+    app.use(express.static(path.join(__dirname, 'ui')));
```

Comment:
Das sieht weird aus. Kopiert `webpack` das so, dass `ui` neben `index.ts` liegt?

Reply:
webpack transpiliert und bundled die ts dateien + dependencies in ./build/server.js, das CopyWebpackPlugin packt die build sourcen in ./build/ui.

Wenn jetzt der server ausgeführt wird mit `node build/server.js` ist __dirname `./build`.